raiden/ui/app.py

```diff
@@ -198,12 +163,14 @@ def run_app(
     if not parsed_eth_rpc_endpoint.scheme:
         eth_rpc_endpoint = f'http://{eth_rpc_endpoint}'
 
-    web3 = _setup_web3(eth_rpc_endpoint)
-    node_network_id, known_node_network_id = setup_network_id_or_exit(config, network_id, web3)
```

Comment:
This is the weirdest thing ever `setup_network_id_or_exit` checked that `web3.version.network` and `network_id` are equal, and returned them as a tuple, so all three values `node_network_id`,`known_node_network_id`, and `network_id` were always equal